Sources/SafeDICore/Models/TypeDescription.swift

```diff
@@ -121,7 +121,7 @@ public enum TypeDescription: Codable, Hashable, Comparable, Sendable {
         case let .closure(arguments, isAsync, doesThrow, returnType):
             return "(\(arguments.map { $0.asSource }.joined(separator: ", ")))\([isAsync ? " async" : "", doesThrow ? " throws" : ""].filter { !$0.isEmpty }.joined()) -> \(returnType.asSource)"
         case let .unknown(text):
-            return text.trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
since `trimmingCharacters` doesn't exist on Linux, we now initialize this type with a `trimmedDescription` which does effectively the same thing (and we've got a test to prove it!)